gcloud/bigquery/test_client.py

```diff
@@ -74,27 +74,11 @@ def test_list_datasets_defaults(self):
         self.assertEqual(req['method'], 'GET')
         self.assertEqual(req['path'], '/%s' % PATH)
 
-    def test_list_datasets_explicit(self):
```

Comment:
I saw that there was no need for a separate test:  the explicit one was redundant with the defaults one for testing the unmarshalling of the `Dataset` objects, so I just updated to return a set with no `datasets` key at all.